pkg/types/urlsmap.go

```diff
@@ -25,9 +25,12 @@ type URLsMap map[string]URLs
 
 // NewURLsMap returns a URLsMap instantiated from the given string,
 // which consists of discovery-formatted names-to-URLs, like:
-// mach0=http://1.1.1.1:2380,mach0=http://2.2.2.2::2380,mach1=http://3.3.3.3:2380,mach2=http://4.4.4.4:2380
+// mach0=http://1.1.1.1:2380,mach1=http://2.2.2.2::2380,mach2=http://3.3.3.3:2380,mach3=http://4.4.4.4:2380
```

Comment:
i think the two mach0s is intended. we want to show that we do accept duplicate key but different urls.

Probably a better example could be 
```
mach0=http://1.1.1.1:2380,mach0=http://1.1.1.2::2380,mach1=http://2.2.2.2:2380,mach2=http://3.3.3.3:2380
```